sql/payday.sql

```diff
@@ -186,13 +186,14 @@ CREATE OR REPLACE FUNCTION process_payment_instruction() RETURNS trigger AS $$
               FROM payday_participants p
              WHERE username = NEW.participant
         );
+
         IF (NEW.amount + NEW.due <= participant.new_balance OR participant.card_hold_ok) THEN
             EXECUTE pay(NEW.participant, NEW.team, NEW.amount + NEW.due, 'to-team');
             RETURN NEW;
-        ELSE
+        ELSIF NEW.amount + NEW.due <= (SELECT minimum_charge FROM settings) THEN
```

Comment:
> We should never increase dues beyond the minimum charge. If they reach the minimum charge and then their card fails, we should stop adding to due. If/when they fix their card we should charge the minimum at that point. We don't want $260 surprises!

I mean, I guess it makes sense. The reason we are parking dues is to build up to the minimum charge, and once that threshold is reached, we charge the user. In the ideal failure case (modulo #3506), they get an email notification about the failure, fix their card, and get back on track the next week.